bundles/org.openhab.core.thing/src/main/java/org/eclipse/smarthome/core/thing/profiles/SystemProfiles.java

```diff
@@ -50,6 +51,11 @@
             .withSupportedItemTypes(CoreItemFactory.NUMBER).withSupportedItemTypesOfChannel(CoreItemFactory.NUMBER)
             .build();
 
+    TriggerProfileType RAWBUTTON_ON_OFF_SWITCH_TYPE = ProfileTypeBuilder
+            .newTrigger(RAWBUTTON_ON_OFF_SWITCH, "Raw Button On Off Switch")
+            .withSupportedItemTypes(CoreItemFactory.SWITCH)
```

Comment:
Should we add support for `Dimmer` and `Color` items too as they both accept `OnOffType`.
```suggestion
            .withSupportedItemTypes(CoreItemFactory.SWITCH, CoreItemFactory.DIMMER, CoreItemFactory.COLOR)
```

Reply:
I wasn't sure about this. But, I think you're right that they should be added.